e2e-testing/cypress/functions/SurveyConfigurationFunctions.js

```diff
@@ -91,6 +97,29 @@ class SurveyConfigurationFunctions {
     cy.get('.post-info__username').contains('Anonymous').should('be.visible');
   }
 
+  check_for_time_post_was_added() {
```

Comment:
How I checked for this is:
 - where a post has time hidden, the admin would be able to see the exact time a post was sent in, while a non-admin or user without the right privileges will see an inaccurate time
 - the check here then is log in as admin, see the time in the most recently sent in post, and log out and verify that the time seen while logged out is not the same.

Reply:
@AmTryingMyBest this clarification actually makes sense to implement this way.

Reply:
@shakirandagire follow this for hiding location test too